Xamarin.Forms.Core/IPlatformServices.cs

```diff
@@ -33,6 +33,8 @@ public interface IPlatformServices
 
 		string RuntimePlatform { get; }
 
+		IDispatcher GetDispatcher();
```

Comment:
Unfortunately, we can't just add a new method to this interface; because it's public, someone may have implemented their own version and any changes will be breaking. 

So we'll need to move this method to it's own interface. I'd suggest something like `IDispatcherProvider`. Then it can be resolved by the `DependencyService` (take a look how `ISystemResources` is handled in the `Application` constructor).